app/helpers/hyrax_helper.rb

```diff
@@ -4,6 +4,7 @@ module HyraxHelper
   include ::BlacklightHelper
   include Hyrax::BlacklightOverride
   include Hyrax::HyraxHelperBehavior
+  prepend IiifPrint::IiifHelperDecorator
```

Comment:
are you lost little guy? do you need help finding your mommy or daddy pr?

Reply:
not a big deal - just making sure this should be here

Reply:
This little guy did sneak in.  And I'll work on removing it.